### PR TITLE
KAFKA-3983 - Add additional information to debug

### PR DESCRIPTION
**Pull request by rnpridgeon**
Add additional information to Acceptor debug message upon connection acceptance

### CODE REVIEW

File: core/src/main/scala/kafka/network/SocketServer.scala [hunk 1]
```diff
@@ -332,8 +332,9 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
       if (sendBufferSize != Selectable.USE_DEFAULT_BUFFER_SIZE)
         socketChannel.socket().setSendBufferSize(sendBufferSize)
 
-      debug("Accepted connection from %s on %s. sendBufferSize [actual|requested]: [%d|%d] recvBufferSize [actual|requested]: [%d|%d]"
-            .format(socketChannel.socket.getInetAddress, socketChannel.socket.getLocalSocketAddress,
+      debug("Accepted connection from %s on %s. Registering with Processor %d sendBufferSize [actual|requested]: [%d|%d] recvBufferSize [actual|requested]: [%d|%d]"
```
Comment (ijuma) [inline core/src/main/scala/kafka/network/SocketServer.scala]: Maybe:

"Accepted connection from %s on %s and assigned it to processor %d, sendBufferSize..."

What do you think?

File: core/src/main/scala/kafka/network/SocketServer.scala [hunk 2]
```diff
@@ -332,8 +332,9 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
...
+            .format(socketChannel.socket.getRemoteSocketAddress, socketChannel.socket.getLocalSocketAddress,
+                  processor.id,
```
Comment (ijuma) [inline core/src/main/scala/kafka/network/SocketServer.scala]: We can probably have this on the previous line (it's still below the line limit).

File: core/src/main/scala/kafka/network/SocketServer.scala (hunk 1 above)
Comment (rnpridgeon) [inline core/src/main/scala/kafka/network/SocketServer.scala]: works for me, pushing the change now

File: core/src/main/scala/kafka/network/SocketServer.scala (hunk 2 above)
Comment (rnpridgeon) [inline core/src/main/scala/kafka/network/SocketServer.scala]: sure, don't see why not

**Comment by ijuma** [comment]
Thanks for the PR, LGTM.

**Comment by ijuma** [comment]
Merged to trunk and 0.10.0 branches.

**Pull request closed without merging**